util/http/forwarders.go

```diff
@@ -1,48 +0,0 @@
-package http
-
-import (
-	"encoding/json"
-	"fmt"
-	"net/http"
-
-	"github.com/golang/protobuf/proto"
-	"github.com/grpc-ecosystem/grpc-gateway/runtime"
-	"golang.org/x/net/context"
-)
-
-type sseMarshaller struct {
-	runtime.Marshaler
-}
-
-func (m *sseMarshaller) ContentType() string {
-	return "text/event-stream"
-}
-
-func (m *sseMarshaller) Marshal(v interface{}) ([]byte, error) {
-	dataBytes, err := json.Marshal(v)
-	if err != nil {
-		return nil, err
-	}
-	return []byte(fmt.Sprintf("data: %s \n\n", string(dataBytes))), nil
-}
-
-var (
-	StreamForwarder = func(
```

Comment:
Im assuming this was a duplicate file?

Reply:
@alexec - what do you mean? this is a deleted file